go/kbfs/libkbfs/block_retrieval_queue.go

```diff
@@ -266,41 +267,55 @@ func (brq *blockRetrievalQueue) notifyWorker(priority int) {
 	brq.sendWork(workerCh)
 }
 
-func (brq *blockRetrievalQueue) initPrefetchStatusCacheLocked() {
+func (brq *blockRetrievalQueue) initPrefetchStatusCacheLocked() error {
 	if brq.prefetchStatusForTest != nil {
-		return
+		return nil
 	}
 	if !brq.config.IsTestMode() && brq.config.Mode().Type() != InitSingleOp {
 		// If the disk block cache directory can't be accessed due to
 		// permission errors (happens sometimes on iOS for some
 		// reason), we might need to rely on this in-memory map.
-		// TODO(KBFS-3750): make it an LRU cache.
 		brq.log.Warning("No disk block cache is initialized when not testing")
 	}
 	brq.log.CDebugf(nil, "Using a local cache for prefetch status")
-	brq.prefetchStatusForTest = make(map[kbfsblock.ID]PrefetchStatus)
+	// TODO: choose a value for this size
+	var err error
+	brq.prefetchStatusForTest, err = lru.New(1000)
+	return err
 }
 
 func (brq *blockRetrievalQueue) getPrefetchStatus(
-	id kbfsblock.ID) PrefetchStatus {
+	id kbfsblock.ID) (PrefetchStatus, error) {
 	brq.prefetchStatusLock.Lock()
 	defer brq.prefetchStatusLock.Unlock()
 	if brq.prefetchStatusForTest == nil {
-		brq.initPrefetchStatusCacheLocked()
+		err := brq.initPrefetchStatusCacheLocked()
```

Comment:
I know this is not from this PR, but it seems we don't need the `brq.prefetchStatusForTest == nil` check above anymore as we already do that inside the method.